### ISSUE TITLE
AttributeError: 'module' object has no attribute '_strptime' in pool.py

### ISSUE THREAD

**Issue by bousch** [andrey-pohilko/registry-cli#53]
The error below occurs since the threadpool introduction:

```
tag not found
Traceback (most recent call last):
  File "/registry.py", line 790, in <module>
    main_loop(args)
  File "/registry.py", line 759, in main_loop
    args.keep_by_hours, tags_list))
  File "/registry.py", line 655, in get_newer_tags
    result = list(x for x in p.map(newer, tags_list) if x)
  File "/usr/local/lib/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/usr/local/lib/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
AttributeError: 'module' object has no attribute '_strptime'
```

Retrying the same command against the same registry sometimes fails and sometimes works. Is anybody else experiencing the same issue?

**Comment by andrey-pohilko** [comment]
What version of python are you running?

Dne čt 13. 9. 2018 14:33 uživatel bousch <notifications@github.com> napsal:

**Comment by bousch** [comment]
Sorry, should have mentioned that I use the anoxis/registry-cli docker image from docker hub so the version is the one in python:2.7-alpine which is Python 2.7.15.

**Comment by andrey-pohilko** [comment]
Hey @bousch . Can you please check if the issue has been resolved with pull request #54?

**Comment by bousch** [comment]
Thanks, initial tests indicate this has solved the issue. Will know more after our nightly cleanup job has run a couple of times.
